tests/nested/manual/cloud-init-never-used-not-vuln/task.yaml

```diff
@@ -96,16 +96,10 @@ execute: |
     nested_exec "cloud-init status --wait"
 
     echo "Waiting for snapd to react to cloud-init"
-    for i in $(seq 60); do
-        if nested_exec "journalctl --no-pager -u snapd" | MATCH "Cloud-init reported"; then
-            break
-        fi
-        sleep 1
-    done
-
+    nested_exec "retry --wait 1 -n 60 sh -c 'journalctl --no-pager -u snapd | MATCH \"Cloud-init reported\"'"
+    
     # ensure that snapd disabled cloud-init with the cloud-init.disabled file
     echo "Ensuring that snapd restricted cloud-init"
-    nested_exec "cloud-init status" | MATCH "status: disabled"
```

Comment:
please leave this line in, it is an orthogonal check from the above one about "Cloud-init reported as disabled" in snapd journal

Reply:
Sure, it was my mistake.
Thanks for the review